docs/rule/require-each-key.md

```diff
@@ -0,0 +1,54 @@
+# require-each-key
+
+You might be having huge performance issues when rendering large data-sets with `{{#each}}`
```

Comment:
We were having huge performance issues (up to more than a second blocking time for a single click event) when rendering only 10 table rows. I don't think that counts as "large" :)

Reply:
Changed the text accordingly